versions.json

```
@@ -13,6 +13,6 @@
   },
   "1.5.0": {
     "katana": ["1.5.0"],
-    "torii": ["1.5.0"]
-  }
+    "torii": ["1.5.1"]
```

```suggestion
    "torii": ["1.5.1", "1.5.0"]
```
For now we keep the list, but soon we will have something like `1.5` which will ensure we don't have to update this file at each patch. 👍